CONTRIBUTING.md

```diff
@@ -82,19 +82,12 @@ the pull request affects. Valid areas as:
   - *Consensus* for changes to consensus critical code
   - *Docs* for changes to the documentation
   - *Qt* for changes to bitcoin-qt
+  - *Logs* Changes to log messages
```

Comment:
Good addition. Not a strong opinion, but while here, perhaps shorten to log as well as docs to doc and tests to test... since they are shorter and often used in practice. For brevity, scripts and tools could be just script or tool.